src/controls/shared/WidgetWithLabelUI.tsx

```diff
@@ -32,7 +32,7 @@ export const WidgetWithLabelUI = observer(function WidgetWithLabelUI_(p: {
     const { WidgetLineUI, WidgetBlockUI } = WidgetDI.WidgetUI(widget) // WidgetDI.WidgetUI(widget)
 
     const isCollapsible: boolean = checkIfWidgetIsCollapsible(widget)
-    const isCollapsed = widget.serial.collapsed && isCollapsible
+    const isCollapsed = (widget.serial.collapsed ?? isDisabled) && isCollapsible
```

Comment:
Can we still unfolded disabled widget to look at values without activating them ?
It's probably related to your diff in widget_group, and I think it may be simpler / preferable to just change the default active value in optional instead. I'll check later roday